packages/react-core/src/components/EmptyState/EmptyStateActions.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from "@patternfly/react-styles/css/components/EmptyState/empty-state";
+
+export interface EmptyStateActionsProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the EmptyStateActions */
```

Comment:
Nit. Instead can we say `Content rendered inside the empty state actions`.  It is our practice to have our descriptions written in sentence case.